go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/mitchellh/go-wordwrap v0.0.0-20150314170334-ad45545899c7
 	github.com/sergi/go-diff v1.0.0
 	github.com/spf13/pflag v1.0.2
-	github.com/zclconf/go-cty v1.12.1
+	github.com/zclconf/go-cty v1.13.0
```

Comment:
should we bump this to the latest patch?

Reply:
(We could still do it, but I'm inclined to make that a separate change since this particular change doesn't need it.)